metricbeat/module/aerospike/aerospike_test.go

```diff
@@ -96,3 +99,102 @@ func TestParseInfo(t *testing.T) {
 		assert.Equal(t, test.expected, result, test.Name)
 	}
 }
+
+func getStringPointer(value string) *string {
+	return &value
+}
+
+func getBoolPointer(value bool) *bool {
+	return &value
+}
+
+func TestParseClientPolicy(t *testing.T) {
+	sampleUser := "Test"
+	samplePassword := "MySecretPassword"
+	sampleClusterName := "TestCluster"
+
+	UserPasswordClientPolicy := as.NewClientPolicy()
+	UserPasswordClientPolicy.User = sampleUser
+	UserPasswordClientPolicy.Password = samplePassword
+
+	TLSPolicy := as.NewClientPolicy()
+	TLSPolicy.TlsConfig = &tls.Config{}
```

Comment:
I guess the problem is that tls.Config is initialized with default values (and it probably includes TLSv1.0). Is there a constructor that I can use to get a "default" SSL Configuration for Metricbeat.
 
 I can only think about, changing this piece of code as follows
 ```go
 	tlsconfig, _ := tlscommon.LoadTLSConfig(&tlscommon.Config{Enabled: getBoolPointer(true)})
	TLSPolicy.TlsConfig = tlsconfig.ToConfig()
 ```